Dockerfile-default-rocm

```diff
@@ -48,7 +131,9 @@ RUN /tmp/det_dockerfile_scripts/install_google_cloud_sdk.sh
 # google-api-python-client -> google-api-core -> googleapis-common-protos -> protobuf
 # Horovod cannot build with protobuf > 3.20.x
 # latest google-api-python-client requires protobuf >= 3.20.1
-RUN pip install protobuf==3.20.1
+RUN pip install numpy==1.23.1
```

Comment:
What is `numpy` required for? It's not a direct dependency of `determined`, right?

Reply:
Pytorch among other things.